tket/include/tket/Utils/Expression.hpp

```diff
@@ -28,6 +28,15 @@
 #include "Json.hpp"
 #include "Symbols.hpp"
 
+/** Helper struct for use with std::visit */
```

Comment:
https://en.cppreference.com/w/cpp/utility/variant/visit I found this struct here but maybe something better exists elsewhere in the codebase?